scripts/telegram_ros_bridge

```diff
@@ -141,6 +149,46 @@ class TelegramROSBridge(object):
             self._telegram_updater.bot.send_photo(self._telegram_chat_id,
                                                   photo=StringIO(cv2.imencode('.jpg', cv2_img)[1].tostring()))
 
+    def _telegram_location_callback(self, _, update):
+        """
+        Called when a new telegram Location is received. The method will verify whether the incoming Location is
+        from the bridges telegram conversation by comparing the chat_id.
+        :param update: Received update that holds the chat_id and message data
+        """
+        location = update.message.location
+        if isinstance(location, Location):
+            rospy.loginfo("Incoming location from telegram: {}".format(location))
+            if self._telegram_chat_id is None:
+                rospy.logwarn("Discarding message. No active chat_id.")
```

Comment:
Duplicated code .. Maybe we should refactor this in another PR

Reply:
https://github.com/tue-robotics/telegram_ros/issues/2